frontends/web/src/containers/SubmitModel.js

```diff
@@ -0,0 +1,194 @@
+                    functions, classes, or variables as you consider necessary,
+                    just keeping in mind that the final result will be contained
+                    in the "single_evaluation" function of the "ModelController"
+                    class.
+                  </p>
+                </li>
+                <li>
+                  <strong>Where put your model and dependencies</strong>
+                  <p>
+                    To store the different components of the model (weights,
+                    tokenizers, processors, etc) there is a folder called
+                    resources. Remember to put all the libraries necessary for
+                    your model in requirements.txt
+                  </p>
+                </li>
+                <li>
+                  <strong>Test your model</strong>
+                  <p>
+                    To make sure everything is working as intended, you have to
+                    run the following commands:
+                    <ul>
+                      <li>python3 -m venv env</li>
+                      <li>source env/bin/activate</li>
+                      <li>python3 -m pip install -r requirements.txt</li>
+                      <li>python3 -m uvicorn app.app:app --reload</li>
+                      <li>test the single_evaluation method</li>
+                    </ul>
+                  </p>
+                  <div className="m-3">
+                    <input id="input-file" className="d-none" type="file" />
+                    {loadingFile ? (
+                      <Row>
+                        <Col className="text-center">
+                          <center>
+                            <Button
+                              variant="primary"
+                              onClick={handleUploadModel}
+                            >
+                              <i className="fas fa-edit"></i> Upload model
+                            </Button>
+                          </center>
+                        </Col>
+                      </Row>
+                    ) : (
+                      <Row>
+                        <Col className="text-center">
+                          <center>
+                            <Button
+                              variant="primary"
+                              onClick={handleSubmitModel}
+                            >
+                              <i className="fas fa-edit"></i> Submit model
+                            </Button>
+                          </center>
+                        </Col>
+                      </Row>
+                    )}
+                  </div>
+                </li>
+              </ul>
+            </div>
+            <div className="text-center">
+              <h2 className="home-cardgroup-header d-block font-weight-light mb-4 text-uppercase text-reset">
+                Have a question?
+              </h2>
+              <a href="mailto:dynabench@fb.com">
```

Comment:
Hrm I think we should change this email (and others?) to mlcommons.org?

Reply:
let me create another PR only related with this